<!DOCTYPE html> <html lang="en"> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Publications | Jiahui Chen</title> <meta name="author" content="Jiahui Chen"/> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "/> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="/assets/img/uark.png"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://jiahuic.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Jiahui </span>Chen</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">Blog</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">Projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching and Mentoring</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Submenus</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/cv/">CV</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/talks/">Talks</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description"></p> </header> <article> <p>Check <a href="/projects/">Projects</a> for more detail.</p> <div class="publications"> <h2 class="year">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ProtSci</abbr></div> <div id="xu2025pldtree" class="col-sm-8"> <div class="title">PLD-Tree: Persistent Laplacian Decision Tree for Protein-Protein Binding Free Energy Prediction</div> <div class="author"> X. Xu, C. Wang, G.W. Wei, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'J. Chen' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Protein Sciences</em> 2025 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CIS</abbr></div> <div id="walden2025dimensionality" class="col-sm-8"> <div class="title">Dimensionality reduction for k-means clustering of large-scale influenza mutation datasets</div> <div class="author"> E. Walden, J. Chen, and G.W. Wei</div> <div class="periodical"> <em>Communication in Information and Systems</em> 2025 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="year">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CBM</abbr></div> <div id="wee2024integration" class="col-sm-8"> <div class="title">Integration of persistent Laplacian and pre-trained transformer for protein</div> <div class="author"> J. Wee, J. Chen, K. Xia, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'G-W. Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Computers in Biology and Medicine</em> 2024 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">SIAM Rev</abbr></div> <div id="ribando2024graph" class="col-sm-8"> <div class="title">Graph Combinatorial and Hodge Laplacians: Similarity and Difference</div> <div class="author"> E. Ribando-Gros, R. Wang, J. Chen, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'G-W. Wei, Y. Tong' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>SIAM Review</em> 2024 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="year">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JCP</abbr></div> <div id="chen2023cartesian" class="col-sm-8"> <div class="title">A Cartesian FMM-accelerated Galerkin boundary integral Poisson-Boltzmann solver</div> <div class="author"> <em>Jiahui Chen</em>, Johannes Tausch, and Weihua Geng</div> <div class="periodical"> <em>Journal of Computational Physics</em> 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0021999123000761" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>The Poisson-Boltzmann model is an effective and popular approach for modeling solvated biomolecules in continuum solvent with dissolved electrolytes. In this paper, we report our recent work in developing a Galerkin boundary integral method for solving the linear Poisson-Boltzmann (PB) equation. The solver has combined advantages in accuracy, efficiency, and memory usage as it applies a well-posed boundary integral formulation to circumvent many numerical difficulties associated with the PB equation and uses an Cartesian Fast Multipole Method (FMM) to accelerate the GMRES iteration. In addition, special numerical treatments such as adaptive FMM order, block diagonal preconditioners, Galerkin discretization, and Duffy’s transformation are combined to improve the performance of the solver, which is validated on benchmark Kirkwood’s sphere and a series of testing proteins.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="wei2023persistent" class="col-sm-8"> <div class="title">persistent topological Laplacian analysis of SARS-CoV-2 variants</div> <div class="author"> Xiaoqi Wei, <em>Jiahui Chen</em>, and Guo-Wei Wei</div> <div class="periodical"> <em>arXiv preprint arXiv:2301.10865</em> 2023 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CBM</abbr></div> <div id="chen2023topological" class="col-sm-8"> <div class="title">Topological deep learning based deep mutational scanning</div> <div class="author"> J. Chen, D.R. Woldring, F. Huang, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'X. Huang, G.W. Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Computers in Biology and Medicine</em> 2023 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JCBC</abbr></div> <div id="wei2023persistentvariants" class="col-sm-8"> <div class="title">Persistent topological Laplacian analysis of SARS-CoV-2 variants</div> <div class="author"> X. Wei, J. Chen, and G.W. Wei</div> <div class="periodical"> <em>Journal of Computational Biophysics and Chemistry</em> 2023 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Small</abbr></div> <div id="nigam2023shape" class="col-sm-8"> <div class="title">Shape Anisotropy Governed High Performance Nanomagnetosol for In Vivo Magnetic Particle Imaging of Lungs</div> <div class="author"> S. Nigam, J. Mohapatra, A. Makela, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'H. Hayat, J. M. Rodriguez, A. Sun, E. Kenyon, N. A. Redman, D. Spence, G. Jabin, B. Gu, M. Ashry, L. F. Sempere, A. Mitra, J. Li, J. Chen, G.-W. Wei, S. Bolin, B. Etchebarne, J. Ping Liu, C. H. Contag, P. Wang' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Small</em> 2023 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="year">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ChemRev</abbr></div> <div id="gao2021methodology" class="col-sm-8"> <div class="title">Methodology-centered review of molecular modeling, simulation, and prediction of SARS-CoV-2</div> <div class="author"> Kaifu Gao, Rui Wang, <em>Jiahui Chen</em>, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Limei Cheng, Jaclyn Frishcosy, Yuta Huzumi, Yuchi Qiu, Tom Schluckbier, Guo-Wei Wei' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Chemical Reviews</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acs.chemrev.1c00965" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Despite tremendous efforts in the past two years, our understanding of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), virus–host interactions, immune response, virulence, transmission, and evolution is still very limited. This limitation calls for further in-depth investigation. Computational studies have become an indispensable component in combating coronavirus disease 2019 (COVID-19) due to their low cost, their efficiency, and the fact that they are free from safety and ethical constraints. Additionally, the mechanism that governs the global evolution and transmission of SARS-CoV-2 cannot be revealed from individual experiments and was discovered by integrating genotyping of massive viral sequences, biophysical modeling of protein–protein interactions, deep mutational data, deep learning, and advanced mathematics. There exists a tsunami of literature on the molecular modeling, simulations, and predictions of SARS-CoV-2 and related developments of drugs, vaccines, antibodies, and diagnostics. To provide readers with a quick update about this literature, we present a comprehensive and systematic methodology-centered review. Aspects such as molecular biophysics, bioinformatics, cheminformatics, machine learning, and mathematics are discussed. This review will be beneficial to researchers who are looking for ways to contribute to SARS-CoV-2 studies and those who are interested in the status of the field.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ACS ID</abbr></div> <div id="wang2022emerging" class="col-sm-8"> <div class="title">Emerging vaccine-breakthrough SARS-CoV-2 variants</div> <div class="author"> Rui Wang, <em>Jiahui Chen</em>, Yuta Hozumi, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Changchuan Yin, Guo-Wei Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>ACS infectious diseases</em> 2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CJIM</abbr></div> <div id="chen2022omicron" class="col-sm-8"> <div class="title">Omicron variant (B. 1.1. 529): infectivity, vaccine breakthrough, and antibody resistance</div> <div class="author"> <em>Jiahui Chen</em>, Rui Wang, Nancy Benovich Gilby, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Guo-Wei Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Journal of chemical information and modeling</em> 2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JPCL</abbr></div> <div id="chen2022omicronba2" class="col-sm-8"> <div class="title">Omicron BA. 2 (b. 1.1. 529.2): High potential for becoming the next dominant variant</div> <div class="author"> <em>Jiahui Chen</em>, and Guo-Wei Wei</div> <div class="periodical"> <em>The journal of physical chemistry letters</em> 2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="ribando2022graph" class="col-sm-8"> <div class="title">Graph and Hodge Laplacians: Similarity and Difference</div> <div class="author"> Emily Ribando-Gros, Rui Wang, <em>Jiahui Chen</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Yiying Tong, Guo-Wei Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>arXiv preprint arXiv:2204.12218</em> 2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="chen2022mathematical" class="col-sm-8"> <div class="title">Mathematical artificial intelligence design of mutation-proof COVID-19 monoclonal antibodies</div> <div class="author"> <em>Jiahui Chen</em>, and Guo-Wei Wei</div> <div class="periodical"> <em>arXiv preprint arXiv:2204.09471</em> 2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="chen2022persistent" class="col-sm-8"> <div class="title">Persistent Laplacian projected Omicron BA. 4 and BA. 5 to become new dominating variants</div> <div class="author"> <em>Jiahui Chen</em>, Yuchi Qiu, Rui Wang, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Guo-Wei Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Computers in Biology and Medicine</em> 2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CIS</abbr></div> <div id="wilson2022computing" class="col-sm-8"> <div class="title">Computing electrostatic binding energy with the TABI Poisson–Boltzmann solver</div> <div class="author"> Leighton Wilson, Jingzhen Hu, <em>Jiahui Chen</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Robert Krasny, Weihua Geng' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Communications in Information and Systems</em> 2022 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JCIM</abbr></div> <div id="chen2022Emerging" class="col-sm-8"> <div class="title">Emerging dominant SARS-CoV-2 variants</div> <div class="author"> <em>Jiahui Chen</em>, Rui Wang, Yuta Hozumi, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Gengzhou Liu, Yuchi Qiu, Xiaoqi Wei, Guo-Wei Wei' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Journal of Chemical Information and Modeling</em> 2022 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">DCDS–B</abbr></div> <div id="chen2021evolutionary" class="col-sm-8"> <div class="title">Evolutionary de Rham-Hodge method</div> <div class="author"> <em>Jiahui Chen</em>, Rundong Zhao, Yiying Tong, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Guo-Wei Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Discrete and Continuous Dynamical Systems Series B</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.aimsciences.org/article/doi/10.3934/dcdsb.2020257" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>The de Rham-Hodge theory is a landmark of the 20th Century’s mathematics and has had a great impact on mathematics, physics, computer science, and engineering. This work introduces an evolutionary de Rham-Hodge method to provide a unified paradigm for the multiscale geometric and topological analysis of evolving manifolds constructed from a filtration, which induces a family of evolutionary de Rham complexes. While the present method can be easily applied to close manifolds, the emphasis is given to more challenging compact manifolds with 2-manifold boundaries, which require appropriate analysis and treatment of boundary conditions on differential forms to maintain proper topological properties. Three sets of unique evolutionary Hodge Laplacians are proposed to generate three sets of topology-preserving singular spectra, for which the multiplicities of zero eigenvalues correspond to exactly the persistent Betti numbers of dimensions 0, 1, and 2. Additionally, three sets of non-zero eigenvalues further reveal both topological persistence and geometric progression during the manifold evolution. Extensive numerical experiments are carried out via the discrete exterior calculus to demonstrate the potential of the proposed paradigm for data representation and shape analysis. To demonstrate the utility of the proposed method, application is considered to the protein B-factor predictions of a few challenging cases for which other existing models do not work well.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JCBC</abbr></div> <div id="chen2021computing" class="col-sm-8"> <div class="title">Computing protein pKas using the TABI Poisson-Boltzmann solver</div> <div class="author"> <em>Jiahui Chen</em>, Jingzhen Hu, Yongjia Xu, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Robert Krasny, Weihua Geng' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Journal of Computational Biophysics and Chemistry</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CPC</abbr></div> <div id="chen2021cyclically" class="col-sm-8"> <div class="title">Cyclically parallelized treecode for fast computations of electrostatic interactions on molecular surfaces</div> <div class="author"> <em>Jiahui Chen</em>, Weihua Geng, and Daniel R Reynolds</div> <div class="periodical"> <em>Computer Physics Communications</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0010465520303672" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>We study the parallelization of a flexible order Cartesian treecode algorithm for evaluating electrostatic potentials of charged particle systems in which particles are located on the molecular surfaces of biomolecules such as proteins. When the well-separated condition is satisfied, the treecode algorithm uses a far-field Taylor expansion to compute O(NlogN) particle–cluster interactions to replace the O(N^2) particle–particle interactions. The algorithm is implemented using the Message Passing Interface (MPI) standard by creating identical tree structures in the memory of each task for concurrent computing. We design a cyclic order scheme to uniformly distribute spatially-closed target particles to all available tasks, which significantly improves parallel load balancing. We also investigate the parallel efficiency subject to treecode parameters such as Taylor expansion order p, maximum particles per leaf N0, and maximum acceptance criterion theta. This cyclically parallelized treecode can solve interactions among up to tens of millions of particles. However, if the problem size exceeds the memory limit of each task, a scalable domain decomposition (DD) parallelized treecode using an orthogonal recursive bisection (ORB) tree can be used instead In addition to efficiently computing the N-body problem of charged particles, our approach can potentially accelerate GMRES iterations for solving the boundary integral Poisson–Boltzmann equation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">FDS</abbr></div> <div id="wang2021hermes" class="col-sm-8"> <div class="title">HERMES: Persistent spectral graph software</div> <div class="author"> Rui Wang, Rundong Zhao, Emily Ribando-Gros, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jiahui Chen, Yiying Tong, Guo-Wei Wei' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Fondations of data science (Springfield, Mo.)</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ChemSci</abbr></div> <div id="chen2021prediction" class="col-sm-8"> <div class="title">Prediction and mitigation of mutation threats to COVID-19 vaccines and antibody therapies</div> <div class="author"> <em>Jiahui Chen</em>, Kaifu Gao, Rui Wang, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Guo-Wei Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Chemical Science</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01203g#!divAbstract" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>Antibody therapeutics and vaccines are among our last resort to end the raging COVID-19 pandemic. They, however, are prone to over 5000 mutations on the spike (S) protein uncovered by a Mutation Tracker based on over 200 000 genome isolates. It is imperative to understand how mutations will impact vaccines and antibodies in development. In this work, we first study the mechanism, frequency, and ratio of mutations on the S protein which is the common target of most COVID-19 vaccines and antibody therapies. Additionally, we build a library of 56 antibody structures and analyze their 2D and 3D characteristics. Moreover, we predict the mutation-induced binding free energy (BFE) changes for the complexes of S protein and antibodies or ACE2. By integrating genetics, biophysics, deep learning, and algebraic topology, we reveal that most of the 462 mutations on the receptor-binding domain (RBD) will weaken the binding of S protein and antibodies and disrupt the efficacy and reliability of antibody therapies and vaccines. A list of 31 antibody disrupting mutants is identified, while many other disruptive mutations are detailed as well. We also unveil that about 65% of the existing RBD mutations, including those variants recently found in the United Kingdom (UK) and South Africa, will strengthen the binding between the S protein and human angiotensin-converting enzyme 2 (ACE2), resulting in more infectious COVID-19 variants. We discover the disparity between the extreme values of RBD mutation-induced BFE strengthening and weakening of the bindings with antibodies and angiotensin-converting enzyme 2 (ACE2), suggesting that SARS-CoV-2 is at an advanced stage of evolution for human infection, while the human immune system is able to produce optimized antibodies. This discovery, unfortunately, implies the vulnerability of current vaccines and antibody drugs to new mutations. Our predictions were validated by comparison with more than 1400 deep mutations on the S protein RBD. Our results show the urgent need to develop new mutation-resistant vaccines and antibodies and to prepare for seasonal vaccinations.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CommBio</abbr></div> <div id="wang2021analysis" class="col-sm-8"> <div class="title">Analysis of SARS-CoV-2 mutations in the United States suggests presence of four substrains and novel variants</div> <div class="author"> Rui Wang, <em>Jiahui Chen</em>, Kaifu Gao, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Yuta Hozumi, Changchuan Yin, Guo-Wei Wei' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Communications biology</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">AppSci</abbr></div> <div id="strubbe2021modeling" class="col-sm-8"> <div class="title">Modeling the Effects of Calcium Overload on Mitochondrial Ultrastructural Remodeling</div> <div class="author"> Jasiel O Strubbe-Rivera, <em>Jiahui Chen</em>, Benjamin A West, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Kristin N Parent, Guo-Wei Wei, Jason N Bazil' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Applied Sciences</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CIS</abbr></div> <div id="chen2021sars" class="col-sm-8"> <div class="title">SARS-CoV-2 becoming more infectious as revealed by algebraic topology and deep learning</div> <div class="author"> <em>Jiahui Chen</em>, Rui Wang, and Guo-Wei Wei</div> <div class="periodical"> <em>Communications in Information and Systems</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Genomics</abbr></div> <div id="wang2021vaccine" class="col-sm-8"> <div class="title">Vaccine-escape and fast-growing mutations in the United Kingdom, the United States, Singapore, Spain, India, and other COVID-19-devastated countries</div> <div class="author"> Rui Wang, <em>Jiahui Chen</em>, Kaifu Gao, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Guo-Wei Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Genomics</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JMB</abbr></div> <div id="chen2021revealing" class="col-sm-8"> <div class="title">Revealing the threat of emerging SARS-CoV-2 mutations to antibody therapies</div> <div class="author"> <em>Jiahui Chen</em>, Kaifu Gao, Rui Wang, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Guo-Wei Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Journal of molecular biology</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">arXiv</abbr></div> <div id="chen2021review" class="col-sm-8"> <div class="title">Review of the mechanisms of SARS-CoV-2 evolution and transmission</div> <div class="author"> <em>Jiahui Chen</em>, Rui Wang, and Guo-Wei Wei</div> <div class="periodical"> <em>ArXiv</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">CJCP</abbr></div> <div id="chen2021mlimc" class="col-sm-8"> <div class="title">MLIMC: Machine learning-based implicit-solvent Monte Carlo</div> <div class="author"> <em>Jiahui Chen</em>, Weihua Geng, and Guo-Wei Wei</div> <div class="periodical"> <em>Chinese Journal of Chemical Physics</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JPCL</abbr></div> <div id="wang2021mechanisms" class="col-sm-8"> <div class="title">Mechanisms of SARS-CoV-2 evolution revealing vaccine-resistant mutations in Europe and America</div> <div class="author"> Rui Wang, <em>Jiahui Chen</em>, and Guo-Wei Wei</div> <div class="periodical"> <em>The journal of physical chemistry letters</em> 2021 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JMC</abbr></div> <div id="gao2021perspectives" class="col-sm-8"> <div class="title">Perspectives on SARS-CoV-2 main protease inhibitors</div> <div class="author"> Kaifu Gao, Rui Wang, <em>Jiahui Chen</em>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jetze J Tepe, Faqing Huang, Guo-Wei Wei' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Journal of Medicinal Chemistry</em> 2021 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ACS</abbr></div> <div id="gao2020repositioning" class="col-sm-8"> <div class="title">Repositioning of 8565 existing drugs for COVID-19</div> <div class="author"> Kaifu Gao, Duc Duy Nguyen, <em>Jiahui Chen</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Rui Wang, Guo-Wei Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>The journal of physical chemistry letters</em> 2020 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JMB</abbr></div> <div id="chen2020mutations" class="col-sm-8"> <div class="title">Mutations strengthened SARS-CoV-2 infectivity</div> <div class="author"> <em>Jiahui Chen</em>, Rui Wang, Menglun Wang, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Guo-Wei Wei' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Journal of Molecular Biology</em> 2020 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">AR of Biophy.</abbr></div> <div id="chen2020review" class="col-sm-8"> <div class="title">Review of COVID-19 antibody therapies</div> <div class="author"> <em>Jiahui Chen</em>, Kaifu Gao, Rui Wang, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Duc Duy Nguyen, Guo-Wei Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Annual Review of Biophysics</em> 2020 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">MathBio</abbr></div> <div id="zhao2020rham" class="col-sm-8"> <div class="title">The de Rham–Hodge Analysis and Modeling of Biomolecules</div> <div class="author"> Rundong Zhao, Menglun Wang, <em>Jiahui Chen</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Yiying Tong, Guo-Wei Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Bulletin of Mathematical Biology</em> 2020 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JPhysChem</abbr></div> <div id="wang2020decoding" class="col-sm-8"> <div class="title">Decoding asymptomatic COVID-19 infection and transmission</div> <div class="author"> Rui Wang, <em>Jiahui Chen</em>, Yuta Hozumi, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Changchuan Yin, Guo-Wei Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>The journal of physical chemistry letters</em> 2020 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ChemSci</abbr></div> <div id="nguyen2020unveiling" class="col-sm-8"> <div class="title">Unveiling the molecular mechanism of SARS-CoV-2 main protease inhibition from 137 crystal structures using algebraic topology and deep learning</div> <div class="author"> Duc Duy Nguyen, Kaifu Gao, <em>Jiahui Chen</em>, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Rui Wang, Guo-Wei Wei' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Chemical Science</em> 2020 </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="year">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">SMU</abbr></div> <div id="chen2019parallel" class="col-sm-8"> <div class="title">PARALLEL MULTIPOLE EXPANSION ALGORITHMS AND THEIR BIOLOGY APPLICATIONS</div> <div class="author"> <em>Jiahui Chen</em> </div> <div class="periodical"> <em></em> 2019 </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">ProtSci</abbr></div> <div id="jurrus2018improvements" class="col-sm-8"> <div class="title">Improvements to the APBS biomolecular solvation software suite</div> <div class="author"> Elizabeth Jurrus, Dave Engel, Keith Star, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Kyle Monson, Juan Brandi, Lisa E Felberg, David H Brookes, Leighton Wilson, Jiahui Chen, Karina Liles, others' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Protein Science</em> 2018 </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">JCP</abbr></div> <div id="chen2018preconditioning" class="col-sm-8"> <div class="title">On preconditioning the treecode-accelerated boundary integral (TABI) Poisson–Boltzmann solver</div> <div class="author"> <em>Jiahui Chen</em>, and Weihua Geng</div> <div class="periodical"> <em>Journal of Computational Physics</em> 2018 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.sciencedirect.com/science/article/pii/S0021999118304728" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> </div> <div class="abstract hidden"> <p>We recently developed a treecode-accelerated boundary integral (TABI) solver for solving Poisson–Boltzmann (PB) equation [1]. The solver has combined advantages in accuracy, efficiency, memory, and parallelization as it applies a well-posed boundary integral formulation to circumvent many numerical difficulties associated with the PB equation and uses an O(NlogN) treecode to accelerate the GMRES iterative solver. However, as observed in our previous work, occasionally when the mesh generator produces low quality triangles, the number of GMRES iterations required to solve the discretized boundary integral equations Ax=b could be large. To address this issue, we design a preconditioning scheme using preconditioner matrix M such that M^-1A has much improved condition while M^-1z can be rapidly computed for any vector z. In this scheme, the matrix M carries the interactions between boundary elements on the same leaf only in the tree structure thus is block diagonal with many computational advantages. The sizes of the blocks in M are conveniently controlled by the treecode parameter N0, the maximum number of particles per leaf. The numerical results show that this new preconditioning scheme improves the TABI solver with significantly reduced iteration numbers and better accuracy, particularly for protein sets on which TABI solver previously converges slowly. In addition, this preconditioning scheme potentially can improve the condition number of various multipole method accelerated boundary elements solvers in scattering, fluids, elasticity, etc.</p> </div> </div> </div> </li> </ol> <h2 class="year">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="chen2015parallel" class="col-sm-8"> <div class="title">Parallel computing of the adaptive n-body treecode algorithm for solving boundary integral Poisson-Boltzmann equation</div> <div class="author"> <em>Jiahui Chen</em>, and Weihua Geng</div> <div class="periodical"> <em>In International Conference on High Performance Computing and Applications</em> 2015 </div> <div class="links"> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2025 Jiahui Chen. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="noopener noreferrer">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> </body> </html>